person_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../book'
+require_relative '../person'
```

Comment:
- [ ]  @Team Great work implementing the tests for the person class, however, you have imported the book rental and person class wrongly, it should be with a single dot since they are in the same directory eg instead of `require_relative '../book'` it should be `require_relative './book'`